app/src/main/java/com/duckduckgo/app/global/api/PixelParamRemovalInterceptor.kt

```diff
@@ -101,6 +101,11 @@ object PixelInterceptorPixelsRequiringDataCleaning : PixelParamRemovalPlugin {
             AppPixelName.SETTINGS_SYNC_PRESSED.pixelName to PixelParameter.removeAtb(),
             AppPixelName.SETTINGS_EMAIL_PROTECTION_PRESSED.pixelName to PixelParameter.removeAtb(),
             AppPixelName.ONBOARDING_DAX_CTA_DISMISS_BUTTON.pixelName to PixelParameter.removeAtb(),
+            AppPixelName.PREONBOARDING_INTRO_REINSTALL_USER_SHOWN_UNIQUE.pixelName to PixelParameter.removeAtb(),
```

Comment:
NIT: I think adding ATB to these pixels is a good idea (like for a month), it'll help us catch any unexpected cases where this ends up being shown to new users.